js-foundations.js

```diff
@@ -32,19 +94,57 @@ var testString2 = "academy"
 var testString3 = "sandiego"
 
 // 4a. Write a function called alphabetSoup that takes one argument and returns a string with all the letters in alphabetical order. Use each of the varibales as test cases. Expected output: "aelnr" "aacdemy" "adeginos"
+var testString1 = "learn"
+var testString2 = "academy"
+var testString3 = "sandiego" 
+function AlphabetSoup(str) { 
+  newArr = str.match(/[a-z]/g);
+  console.log(newArr);
+  return newArr.sort().join("");
+}
 
+ AlphabetSoup(testString1, testString2, testString3); 
 
 
 // 4b. Write a function that takes in all three variables and returns a string with all the letters in alphabetical order. Expected output: "aaaacddeeegilmnnorsy"
 
-
+var testString1 = "learn"
+var testString2 = "academy"
+var testString3 = "sandiego"
+function alphabet_order(str)
+  {
+return str.split('').sort().join('');
+  }
+console.log(alphabet_order(testString1));
+console.log(alphabet_order(testString2));
+console.log(alphabet_order(testString3));
 
```

Comment:
This works great for sorting each string. The prompt is asking for all the letters sorted in one string.